### PR TITLE
Set repeater rate limit delay in settings.py

### PR DESCRIPTION
**Pull request by gherceg**
## Technical Summary

Would be easier to tune these values if they lived in settings instead of being hard coded.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/motech/repeaters/const.py [hunk 1]
```diff
@@ -6,7 +6,10 @@
 
 MAX_RETRY_WAIT = timedelta(days=7)
 MIN_RETRY_WAIT = timedelta(minutes=60)
-RATE_LIMITER_DELAY_RANGE = (timedelta(minutes=0), timedelta(minutes=15))
+RATE_LIMITER_DELAY_RANGE = (
+    timedelta(minutes=settings.MIN_REPEATER_RATE_LIMIT_DELAY),
+    timedelta(minutes=settings.MAX_REPEATER_RATE_LIMIT_DELAY),
+)
```
Comment (mjriley) [inline corehq/motech/repeaters/const.py:12]: I'm curious how you feel about defaults? When we reference these values directly, they need to be present in our `settings` object or an error is thrown. I've been looking at using syntax like `getattr(settings, 'MAX_REPEATER_RATE_LIMIT_DELAY', 15)` instead, as it avoids the need to put everything in settings.  At a certain point, when we place everything in settings, I'd imagine the settings file becomes pretty overwhelming. Provided we have sensible defaults, using `getattr` reduces the overall settings size at the cost of some discoverability.

I wonder if the divide is between settings the user will know they need to tweak without any knowledge of our system (i.e. custom URLs) vs technical details. Not sure where this should fall.

**Review by mjriley — APPROVED** [review]
Approving -- left a non-blocking comment

File: corehq/motech/repeaters/const.py (hunk 1 above)
Comment (gherceg) [reply corehq/motech/repeaters/const.py:12]: Yeah I like the sound of that better. I would think this is a technical detail that users will likely never need to think about so going with your approach seems objectively better.

Comment (gherceg) [reply corehq/motech/repeaters/const.py:12]: > the cost of some discoverability.

I suppose if we were really concerned about this, we could create some helper function to do exactly this, but just makes it easy to grep for. Though for now, I'm not too concerned about that.

Comment (gherceg) [reply corehq/motech/repeaters/const.py]: 52d49d8cf0a6e516b531a7e696cf9a28dd98e62f

**Review by mjriley — APPROVED** [review]

**Pull request merged**